frontend/src/features/public-form/utils/createSubmission.ts

```diff
@@ -166,17 +166,21 @@ const encryptAttachment = async (
   attachment: File,
   { id, publicKey }: { id: string; publicKey: string },
 ): Promise<StorageModeAttachment & { id: string }> => {
-  const fileArrayBuffer = await attachment.arrayBuffer()
-  const fileContentsView = new Uint8Array(fileArrayBuffer)
+  try {
+    const fileArrayBuffer = await attachment.arrayBuffer()
+    const fileContentsView = new Uint8Array(fileArrayBuffer)
 
-  const encryptedAttachment = await formsgSdk.crypto.encryptFile(
-    fileContentsView,
-    publicKey,
-  )
-  const encodedEncryptedAttachment = {
-    ...encryptedAttachment,
-    binary: encodeBase64(encryptedAttachment.binary),
+    const encryptedAttachment = await formsgSdk.crypto.encryptFile(
+      fileContentsView,
+      publicKey,
+    )
+    const encodedEncryptedAttachment = {
+      ...encryptedAttachment,
+      binary: encodeBase64(encryptedAttachment.binary),
+    }
+    return { id, encryptedFile: encodedEncryptedAttachment }
+  } catch (error) {
+    console.error(error)
+    return { id, encryptedFile: undefined }
```

Comment:
Hmm, do we really want to do that? Does that create an artificial success for submission, while missing the attachment?

If we catch the error, it might not be reported to DD any more? I thought we would only rethrow 🤔 .

Also if there is an exception, and we assume it's `arrayBuffer is not a function`, we will want to print more information about the context of the error, not just the error (which we already have in DD). If `attachment` is a `File` (per TS definition above), then it *should* have a `.arrayBuffer()` method. If there's none, what else will help us figure out what's wrong? I'd probably add `console.error(attachment)`, and maybe even print  `typeof attachment` for good measure. And anything else you could think of?